packages/cli/src/utils/deploy-v2.ts

```diff
@@ -115,15 +123,28 @@ export async function deploy(
   // Create World contract instance from deployed address
   const WorldContract = new ethers.Contract(await contractPromises.World, IBaseWorldData.abi, signer) as IBaseWorld;
 
+  let confirmations = 1;
+  const retryCount = 0;
+  if (disableTxWait) {
+    confirmations = 0;
+  }
```

Comment:
```suggestion
  const confirmations = disableTxWait ? 0 : 1;
```

also probably worth moving `retryCount` be the last argument in `fastTxExecute` so we don't have to provide a value when calling the method. (`retyCount = 0` should be implicit when calling the method for the first time and only be increased by recursive calls to the method itself)